drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -917,6 +928,7 @@ ShaderCompilerGLES2::ShaderCompilerGLES2() {
 	actions[VS::SHADER_CANVAS_ITEM].usage_defines["NORMAL"] = "#define NORMAL_USED\n";
 	actions[VS::SHADER_CANVAS_ITEM].usage_defines["NORMALMAP"] = "#define NORMALMAP_USED\n";
 	actions[VS::SHADER_CANVAS_ITEM].usage_defines["LIGHT"] = "#define USE_LIGHT_SHADER_CODE\n";
+	actions[VS::SHADER_CANVAS_ITEM].usage_defines["round"] = "#define ROUND_USED\n";
```

Comment:
One question about it,  why don't add usage_defines for spatial shader here?

Reply:
Ah. Good catch. I added just for CanvasItem while testing and then forgot to add to Spatial before comitting.

Reply:
And for Particles too ?

Reply:
This is only for GLES2. So no particle shaders. ;p